__tests__/creatures.test.js

```diff
@@ -0,0 +1,65 @@
+import pool from '../lib/utils/pool.js';
+import setup from '../data/setup.js';
+import request from 'supertest';
+import app from '../lib/app.js';
+import Buffy from '../lib/models/Buffy.js';
+
+describe('character CRUD routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('creates a character via POST', async () => {
+    const character = { name: 'Buffy Summers', species: 'slayer' };
+    const res = await request(app).post('/api/v1/buffy').send(character);
```

Comment:
all your routes in this file should be pointing towards creatures, not buffy (since you've already tested those routes)